src/Lumper.UI/Views/VtfBrowser/VtfBrowserView.axaml.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Markup.Xaml;
+using Lumper.UI.ViewModels.VtfBrowser;
+
+namespace Lumper.UI.Views.VtfBrowser;
+
+public partial class VtfBrowserView : UserControl
+{
+    public VtfBrowserView()
+    {
+        InitializeComponent();
+    }
+
+    private void InitializeComponent()
+    {
+        AvaloniaXamlLoader.Load(this);
+    }
+
+    private void Item_PointerPressed(object? sender, PointerPressedEventArgs e)
+    {
+        if (e.ClickCount == 2)
+        {
+            var selectedVtf = (VtfBrowserViewModel.Vtf)((Border)sender!).DataContext!;
+
+            var window = new VtfImageWindow();
+            window.DataContext = new VtfImageWindowViewModel { Image = selectedVtf.Image, Name = selectedVtf.Name};
+            window.Show();
+        }
```

Comment:
Does `DoubleTapped` not work here? From docs I can't tell if that's just for touchscreen or also works for cursors.

Reply:
seems to work yes. PointerPressed was from stackoverflow, I didn't read the docs on this one